source/user-manual/manager/configuring-cluster/advanced-settings.rst

```diff
@@ -1,27 +1,735 @@
+                    :emphasize-lines: 27
+
+                    #!/usr/bin/env bash
+
+                    tcplog_enabled=$(echo "${TCPLOG:-true}" | tr '[:upper:]' '[:lower:]')
+                    httplog_enabled=$(echo "${HTTPLOG:-true}" | tr '[:upper:]' '[:lower:]')
+                    echo TCPLOG: $tcplog_enabled HTTPLOG: $httplog_enabled
+                    if [ "$tcplog_enabled" = "true" ]; then
+                    sed -i 's/# option tcplog/  option tcplog/' /etc/haproxy/haproxy.cfg
+                    else
+                    sed -i 's/  option tcplog/# option tcplog/' /etc/haproxy/haproxy.cfg
+                    fi
+                    if [ "$httplog_enabled" = "true" ]; then
+                    sed -i 's/# option httplog/  option httplog/' /etc/haproxy/haproxy.cfg
+                    else
+                    sed -i 's/  option httplog/# option httplog/' /etc/haproxy/haproxy.cfg
+                    fi
+                    if [ "$tcplog_enabled" = "true" ] || [ "$httplog_enabled" = "true" ]; then
+                    sed -i 's/#  log /  log /' /etc/haproxy/haproxy.cfg
+                    else
+                    sed -i 's/  log /#  log /' /etc/haproxy/haproxy.cfg
+                    fi
+
+                    # Set env-file
+                    env > /.env-file
+                    # Start HAProxy service
+                    service haproxy start
+                    # Start HAProxy Data Plane API
+                    dataplaneapi -f /etc/haproxy/dataplaneapi.yml &
+
+                    tail -f /dev/null
+
+            4. Build and run the image
+
+                .. code-block:: console
+
+                    # docker build --tag=haproxy-deploy .
+
+                .. code-block:: console
+
+                    # docker run -p 5555:5555 haproxy-deploy
+                    TCPLOG: true HTTPLOG: true
+                    * Starting haproxy haproxy                                                                                                                                                                                          [NOTICE]   (33) : haproxy version is 2.8.9-1842fd0
+                    [NOTICE]   (33) : path to executable is /usr/sbin/haproxy
+                    [ALERT]    (33) : config : parsing [/etc/haproxy/haproxy.cfg:3] : 'pidfile' already specified. Continuing.
+
+            5. Verify the API is running properly
+
+                .. code-block:: console
+
+                    # curl -X GET --user haproxy:haproxy http://localhost:5555/v2/info
+                    {"api":{"build_date":"2024-05-13T14:06:03.000Z","version":"v2.9.3 59f34ea1"},"system":{}}
+
+
+On the Wazuh's side, we will include the ``<haproxy_helper>...</haproxy_helper>`` labels in the :ref:`configuration <haproxy_helper>` file (``/var/ossec/etc/ossec.conf``)  within the ``<cluster>...</cluster>`` section.
+
+We are going to configure a basic HAProxy helper within an already configured cluster master node:
```

Comment:
We should emphasize that this configuration is only necessary on the master node.